### PR TITLE
done review version

### PR DESCRIPTION
**Pull request by vanja-cvetkovic-ey**
work version

### CODE REVIEW

**Comment by vanja-cvetkovic-ey** [comment]
.

File: mission_to_mars/src/components/carousel/Carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      {loading && !err && !activeImgs && <Loader />}
+      {!loading && err && !activeImgs && (
+        <h4 className="error msg">
+          Our messages are still out of the earth <br /> please try again later
+        </h4>
+      )}
+      {!loading && !err && activeImgs && (
+        <>
+          <div className="flex-row">
+            {/* <div className="arrow">
```
Comment (Momir-Radovanovic-EY) [inline mission_to_mars/src/components/carousel/Carousel.jsx:116]: Remove commented code.

File: mission_to_mars/src/components/carousel/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                    onClick={() => dotsHandler(dot)}
+                  ></div>
+                )
+              )}
+            </div>
+          )}
+        </>
+      )}
+      <div className="dots">
+        {/* {dots.map((dot) =>
```
Comment (Momir-Radovanovic-EY) [inline mission_to_mars/src/components/carousel/Carousel.jsx:169]: Remove commented code.

File: mission_to_mars/src/components/carousel/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                <AiOutlineArrowRight />
+              </div>
+            ) : (
+              <div className="arrow disabled">
+                <AiOutlineArrowRight />
+              </div>
+            )}
+          </div>
+          {dots.length > 6 ? (
+            <div className="dots"></div>
```
Comment (Momir-Radovanovic-EY) [inline mission_to_mars/src/components/carousel/Carousel.jsx:150]: you can use like this if there is no children inside it <div />

File: mission_to_mars/src/components/carousel/Carousel.jsx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+            <div className="dots">
+              {dots.map((dot) =>
+                dot === counter ? (
+                  <div key={dot} className="dot active"></div>
+                ) : (
+                  <div
+                    key={dot}
+                    className="dot"
+                    onClick={() => dotsHandler(dot)}
+                  ></div>
```
Comment (Momir-Radovanovic-EY) [inline mission_to_mars/src/components/carousel/Carousel.jsx:161]: Same here

**Review by Momir-Radovanovic-EY — CHANGES_REQUESTED** [review]
Vanja, excellent work! Tested the PR locally, works as expected.

I especially liked how you managed your styles (scss, using variables, mixins, includes etc) inside the app, kudos for that.

Found only one UI bug(which is really impressive) and will share with you some best practices.

**Folder Structuring:**
- Overall good structuring but maby you can improve it a little bit more. Inside pages folder, add folders per pages. For example /pages/Home/ and inside it Home.jsx and Home.scss
- Also that for layout components
- Inside carousel folder component you have unused functioanl Dot.jsx
- In assets folder you will always see files with .svg .png .jpeg extensions but never jsx and scss. - Also you will not see svg as functional components but let it be for now. 

**Carousel:**

On mobile resolution if the image is wider(or something else ), card is outside of a layout
image

- On first rendering Carousel Cards are loaded with first data that came from api, after few seconds it loads again with different data? Not sure why but please check that

**Other:**

- All static text that can be translated, put in constants and pull the value from there(for example your lorem ipsum on other pages, button , link text...)
- Clear all console.log-s inside the app
- Clear all errors/warnings in console
image
- Clear all commented/unused codes
- Do research on async/await and try/catch block
- Logo Animation 👍 
- Css/Scss, responsive, using mixings, includes etc. 🥇 
- If you want to improve it more, you can all CSS properties set to be in Alphabetical order inside that block scope(not mandatory but good practice)

Also resolve requested changes below, before merging.

Over all good job once again 🚀

**Review by Momir-Radovanovic-EY — CHANGES_REQUESTED** [review]
Hi Vanja, very good changes!
Still there is small adjustments that I want from you to do:

- Try to reorganize your imports better, group them by external/local imports and divide them by new line.
-Put all static text inside constants for translation purposes. And after import use it like: <p>{some_content}</p>

-Delete all console.logs
-Write README.md file, what the project is about and how it is used

Go through my comments inside PR and resolve that as well.

**Comment by Momir-Radovanovic-EY** [comment]
Well done! 🥇

**Pull request merged**